### ISSUE TITLE
remove an extra dash from numpy.org website's title "NumPy - "

### ISSUE THREAD

**Issue by partev** [numpy/numpy.org#766]
current Numpy.org's website title is "NumPy - "

It is not clear what the purpose of the extra dash is. It feels like something else, like a short description of NumPy was meant to follow after the dash but is somehow missing. I suggest either removing the dash, or adding a short description after the dash.

![Screenshot 2024-07-22 01 08 29](https://github.com/user-attachments/assets/6fa7d91f-b105-41c9-91e5-5211c081217b)

**Comment by mattip** [comment]
I think this comes from the scientific-python hugo theme, from [here](https://github.com/scientific-python/scientific-python-hugo-theme/blob/827e00a1ec9b059cd21f8f5885e7c71ae561478a/layouts/_default/baseof.html#L6)
```
 <title>{{ block "title" . }}{{ .Site.Title }}{{ if not (eq .Site.Title .Title) }} - {{ .Title }}{{end}}{{ end }}</title>
```

Perhaps the main page does not have a `.Title` attribute. @stefanv thoughts?

**Comment by Musharraffaijaz** [comment]
@mattip can you p[lease check and verify the commit
 fixed the page title issue #766 and removed the redundant hyphen #658

**Comment by stefanv** [comment]
Yes, we'll have to check why "if not (eq .Site.Title .Title)" triggers.

**Comment by stefanv** [comment]
Problem is that `Title` is defined for the index page but empty: https://raw.githubusercontent.com/numpy/numpy.org/refs/heads/main/content/en/_index.md

Should be unset, I think.

**Comment by Musharraffaijaz** [comment]
> Problem is that `Title` is defined for the index page but empty: https://raw.githubusercontent.com/numpy/numpy.org/refs/heads/main/content/en/_index.md
> 
> Should be unset, I think.

The PR, https://github.com/scientific-python/scientific-python-hugo-theme/pull/658 shows the fails builds. Whenever I have tried some changes in the submodules the build fails, can you please suggest me anything why it happens so?
